src/rosdiscover/recover/loader.py

```diff
@@ -107,6 +107,7 @@ def _load_var_ref(self, dict_: t.Mapping[str, t.Any]) -> SymbolicVariableReferen
         return SymbolicVariableReference(
             variable=dict_["qualified_name"],
             type_=type_,
+            initial_value=self._load_value(dict_["initialValue"]),
```

Comment:
we should probably stick to kebab case for consistency in the JSON format (i.e., `initialValue` -> `initial-value`)

Reply:
fixed